db/app/app-tests/app.test.js

```diff
@@ -23,16 +23,28 @@ describe("GET /api/topics", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
-      .then(({body}) => {
-        const  {topics} = body
-        expect(topics).toHaveLength(3)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toHaveLength(3);
         topics.forEach((topic) => {
           expect(topic).toMatchObject({
             description: expect.any(String),
             slug: expect.any(String),
           });
-          
         });
       });
   });
 });
+
+describe("GET /api", () => {
+  test("200: should return a description of all other avaliable endpoints with specific endpoints present in data response", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({ endpoints: expect.any(Object) });
+        expect(body.endpoints["GET /api/topics"]).toBeDefined();
+        expect(body.endpoints["GET /api/articles"]).toBeDefined();
+      });
```

Comment:
It would be good if this test didn't need to be updated after every endpoint - you could achieve this by importing the endpoints.json file